### PR TITLE
cam: add page

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
v0.0.5

### CODE REVIEW

**Comment by Managor** [comment]
The documentation on `cam` is extremely lacking since it's just v0.0.5. Manned's `cam` page is some ancient CPU audio mixer https://manned.org/cam.1 and the official page doesn't have any easily searchable mentions of the tool.

**Review by einverne — APPROVED** [review]
LGTM, thanks for the new page.

File: pages/linux/cam.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# cam
+
+> Frontend tool for libcamera.
```
Comment (kbdharun) [inline pages/linux/cam.md:3]:
```suggestion
> Frontend tool for `libcamera`.
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**